ross/defects/misalignment.py

```diff
@@ -66,18 +72,16 @@ class MisalignmentFlex(Defect):
     ----------
     .. [1] 'Xia, Y., Pang, J., Yang, L., Zhao, Q., & Yang, X. (2019). Study on vibration response 
     and orbits of misaligned rigid rotors connected by hexangular flexible coupling. Applied 
-    Acoustics, 155, 286-296..
+    Acoustics, 155, 286-296 ..
 
     Examples
     --------
-    AQUI AINDA TEM QUE SER ATUALIZADO, ABAIXO SEGUE SOMENTE UM EXEMPLO PARA A "SHAFT ELEMENT"
-    >>> from ross.materials import steel
-    >>> Timoshenko_Element = ShaftElement(
-    ...                         material=steel, L=0.5, idl=0.05, odl=0.1,
-    ...                         rotary_inertia=True,
-    ...                         shear_effects=True)
-    >>> Timoshenko_Element.phi
-    0.1571268472906404
+    >>> from ross.defects.misalignment import misalignment_flex_parallel_example
+    >>> probe1 = (14, 0)
+    >>> probe2 = (22, 0)
+    >>> response = rotor.run_defect(misalignment)
+    >>> results = misalignment.run_time_response()
+    >>> response.plot_dfft(probe=[probe1, probe2], range_freq=[0, 100]).show()
```

Comment:
In examples like this one, try avoiding using commands to display figures. Instead, you could do:
```python
>>> from ross.defects.misalignment import misalignment_flex_parallel_example
>>> probe1 = (14, 0)
>>> probe1 = (22, 0)
>>> response = rotor.run_defect(misalignment)
>>> results = misalignment.run_time_response()
>>> fig = response.plot_dfft(probe=[probe1, probe2], range_freq=[0, 100])
```

or, if you want to make it clear how to show figures, insert one last line with a `#` at the beginning:
```python
>>> from ross.defects.misalignment import misalignment_flex_parallel_example
>>> probe1 = (14, 0)
>>> probe1 = (22, 0)
>>> response = rotor.run_defect(misalignment)
>>> results = misalignment.run_time_response()
>>> fig = response.plot_dfft(probe=[probe1, probe2], range_freq=[0, 100])
>>> # fig.show()
```